packages/api-aco/src/subscriptions/aftePageCreate.ts

```diff
@@ -0,0 +1,28 @@
+import { ContextPlugin } from "@webiny/api";
+import WebinyError from "@webiny/error";
+import get from "lodash.get";
+
+import { ACOContext } from "~/types";
+
+export const afterPageCreate = () => {
+    return new ContextPlugin<ACOContext>(async context => {
+        context.pageBuilder.onPageAfterCreate.subscribe(async ({ page, meta }) => {
+            try {
+                const folderId = get(meta, "location.folderId", "ROOT");
+                await context.folders.createLink({ id: page.pid, folderId });
+            } catch (error) {
+                /**
+                 * In case the link already exists for the current page, just return.
+                 */
+                if (error.code === "LINK_EXISTS") {
+                    return;
+                }
+
+                throw WebinyError.from(error, {
+                    message: "Error while creating page link.",
```

Comment:
I would rather include the original error message, from the original `error` object, instead of masking it.

Reply:
[Webiny.from()](https://github.com/webiny/webiny-js/blob/dev/packages/error/src/Error.ts#L28) should already use code and message from the original error, right? 

So, in case these exist already in the original error, we would use them.

Reply:
You are right @leopuleo. Ignore this then.